test/test_math_ops.cpp

```diff
@@ -2164,6 +2164,80 @@ TEST(MathOps, SquaredDifferenceBroadcasting) {
   opexecuter.RunTest();
 }  // end of test op SquaredDifferenceBroadcasting
 
+// Test op: Xdivy
+TEST(MathOps,Xdivy) {
+  Scope root = Scope::NewRootScope();
+  int dim1 = 2;
+  int dim2 = 2;
+
+  Tensor A(DT_FLOAT, TensorShape({dim1, dim2}));
+  Tensor B(DT_FLOAT, TensorShape({dim1, dim2}));
+
+  AssignInputValues(A, 4.3f);
+  AssignInputValues(B, 3.7f);
+
+  vector<int> static_input_indexes = {};
+  auto R = ops::Xdivy(root, A, B);
+
+  vector<DataType> output_datatypes = {DT_FLOAT};
+
+  std::vector<Output> sess_run_fetchoutputs = {R};
+  OpExecuter opexecuter(root, "Xdivy", static_input_indexes,
+                        output_datatypes, sess_run_fetchoutputs);
+
+  opexecuter.RunTest();
+}  
+
+TEST(MathOps,XdivyZeroX) {
+  Scope root = Scope::NewRootScope();
+  int dim1 = 2;
+  int dim2 = 2;
+
+  Tensor A(DT_FLOAT, TensorShape({dim1, dim2}));
+  Tensor B(DT_FLOAT, TensorShape({dim1, dim2}));
+
+  AssignInputValues(A, std::vector<float>{0.f, 1.f, 5.f, 3.21f ,8.1f });
+  AssignInputValues(B, std::vector<float>{2.0f, 1.2f, 4.2f, 8.9f});
```

Comment:
Lets add tests where 
1. num is non zero positive and denom is 0
2. num is non zero negative and denom is 0